### ISSUE TITLE
NVDA does not voiced characters in applications Microsoft

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#6674]
When
write in
Universal
applications
Microsoft nvda does not read the input character. windows 10 version1607
